tests/sweep_framework/architecture.py

```diff
@@ -0,0 +1,12 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+def tt_smi_command(arch: str):
+    if arch == "grayskull":
+        return ["/home/software/syseng/gs/tt-smi", "-tr", "0"]
+    elif arch == "wormhole" or arch == "wormhole_b0":
+        return ["/home/software/syseng/wh/tt-smi", "-wr", "all", "wait"]
+    else:
+        return ["/home/software/syseng/bh/tt-smi", "-wr", "all", "wait"]
```

Comment:
Don't assume this yet. Please put a `raise Exception()` first and then we can work on this later.

Reply:
Fixed